Casks/i/insomnium.rb

```diff
@@ -0,0 +1,29 @@
+cask "insomnium" do
+  version "0.2.3-a"
+  sha256 "3a561fa0d3413d23180d5488b0034a5bbcb2d433e4600e31d002e0ca2a52d9df"
+
+  url "https://github.com/ArchGPT/insomnium/releases/download/core%40#{version}/Insomnium.Core-#{version}.signed.dmg"
+  name "Insomnium"
+  desc "HTTP & GraphQL Client. FOSS Fork of Insomnia. 100%-Local, no user login, etc"
```

Comment:
```suggestion
  desc "HTTP and GraphQL Client"
```
The above is in line with the [`desc` guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).